src/mugs.js

```diff
@@ -1,3 +1,37 @@
+/**
+ * A mug is a question, containing data, bind, and control elements.
+ *
+ * Main Properties
+ *  nodeID
+ *  label
```

Comment:
Oh, I'd meant to add a file-level comment to `src/javaRosa/plugin.js` once the refactor was done, thanks for the reminder. Did that, and added the itext properties there.